src/components/crownConfig.vue

```diff
@@ -207,20 +204,17 @@ export default {
         cursor: 'pointer',
       };
     },
+    paperDoneRendering() {
+      if (!this.isRendering) {
+        return;
+      }
+
+      new Promise(resolve => this.paper.once('render:done', resolve));
+    },
```

Comment:
I feel like this should belong to `paperManager`, but the reliance on `isRendering` makes this tricky, so I'll defer that change for now.

Knowing if rendering was triggered, or if it's actually in progress would help clean up our rendering related logic. I posted this question in the JointJS repo here: https://github.com/clientIO/joint/issues/1219.